packages/editor/package.json

```diff
@@ -4,10 +4,14 @@
   "description": "Theia - Editor Extension",
   "dependencies": {
     "@theia/core": "^0.11.0",
+    "@theia/filesystem": "^0.11.0",
```

Comment:
i don't like it :( There were times when one could use an editor standalone, now we need the whole Theia.

Can we move it to the workspace extension? Why do we need outline extension?

Reply:
Done with commit 03eb644.